state/state.go

```diff
@@ -47,8 +51,11 @@ type State struct {
 	// AppHash is updated after Commit
 	AppHash []byte
 
+	// XXX: do we need this json tag ?
 	TxIndexer txindex.TxIndexer `json:"-"` // Transaction indexer.
 
+	lastHeightValidatorsChanged int
```

Comment:
Why not export this too?

Reply:
Didn't see a reason to. Should it be? It gets persisted with the validators, felt it didn't need to be persisted with the state.